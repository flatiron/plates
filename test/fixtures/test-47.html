<a data-bind-href="zero" data-bind-class="test" class="">link</a>
