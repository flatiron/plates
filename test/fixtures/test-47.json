{
  "zero": "/sub-zero",
  "one": "oneclass"
}
