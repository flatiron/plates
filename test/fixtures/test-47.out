<a data-bind-href="zero" data-bind-class="test" class="oneclass" href="/sub-zero">link</a>
